ModelicaReference/package.mo

```diff
@@ -5529,89 +5529,181 @@ equivalent to equations in the instantiated parent class are discarded.
+  <td>yes</td>
+  <td></td><td></td><td></td><td></td><td></td><td></td><td></td>
   <td>yes</td>
-  <td></td><td></td><td></td><td></td><td></td>
 </tr>
 <tr>
   <td>type</td>
   <td></td>
   <td></td>
+  <td></td>
+  <td></td>
+  <td>yes</td>
+  <td></td><td></td><td></td><td></td><td></td><td></td>
   <td>yes</td>
-  <td></td><td></td><td></td><td></td>
 </tr>
 <tr>
   <td>record</td>
   <td></td>
   <td></td>
   <td></td>
+  <td></td>
+  <td></td>
+  <td>yes</td>
+  <td></td><td></td><td></td><td></td><td></td>
   <td>yes</td>
-  <td></td><td></td><td></td>
 </tr>
 <tr>
-  <td>connector</td>
+  <td>operator<br>record</td>
+  <td></td>
+  <td></td>
+  <td></td>
+  <td></td>
   <td></td>
   <td></td>
   <td>yes</td>
+  <td></td><td></td><td></td><td></td>
   <td>yes</td>
+</tr>
+<tr>
+  <td>exapandable<br>connector</td>
+  <td></td>
+  <td></td>
+  <td></td>
+  <td></td>
+  <td></td>
+  <td></td>
+  <td></td>
+  <td>yes</td>
+  <td></td><td></td><td></td>
+  <td>yes</td>
+</tr>
+<tr>
+  <td>connector</td>
+  <td></td>
+  <td></td>
+  <td></td>
+  <td></td>
+  <td style=\"background-color:lightgray\">yes</td>
+  <td style=\"background-color:lightgray\">yes</td>
+  <td style=\"background-color:lightgray\">yes</td>
+  <td style=\"background-color:lightgray\">yes</td>
   <td>yes</td>
   <td></td><td></td>
+  <td style=\"background-color:lightgray\">yes</td>
 </tr>
 <tr>
   <td>block</td>
   <td></td>
   <td></td>
   <td></td>
-  <td>yes</td>
+  <td></td>
+  <td></td>
+  <td style=\"background-color:lightgray\">yes</td>
+  <td></td>
+  <td></td>
   <td></td>
   <td>yes</td>
   <td></td>
+  <td style=\"background-color:lightgray\">yes</td>
 </tr>
 <tr>
   <td>model</td>
   <td></td>
   <td></td>
   <td></td>
-  <td>yes</td>
   <td></td>
+  <td></td>
+  <td style=\"background-color:lightgray\">yes</td>
+  <td></td>
+  <td></td>
+  <td></td>
+  <td style=\"background-color:lightgray\">yes</td>
   <td>yes</td>
+  <td style=\"background-color:lightgray\">yes</td>
+</tr>
+<tr>
+  <td>class</td>
+  <td></td>
+  <td></td>
+  <td></td>
+  <td></td>
+  <td></td>
+  <td></td>
+  <td></td>
+  <td></td>
+  <td></td>
+  <td></td>
+  <td></td>
   <td>yes</td>
 </tr>
+</tbody>
 </table>
 
-<p>The specialized classes <code>package</code>,  <code>function</code>,  <code>type</code>, and  <code>record</code> can only be derived from their own kind <em>(e.g., a package can only be base class for packages. All other kinds of classes can use the import statement to use the contents of a package)</em>.</p>
+<p>The specialized classes <code>package</code>, <code>operator</code>,
+<code>function</code>, <code>type</code>, <code>record</code>,
+<code>operator record</code> and <code>expandable connector</code>
+can only be derived from their own kind
+[<em>(e.g., a package can only be base class for packages.
+All other kinds of classes can use the import statement to use the contents
+of a package)</em>] and from <code>class</code>.</p>
```

Comment:
Why double parentheses? Why is this text emphasized?

Reply:
That's taken from the specs. So I kept it for now to be in sync.

Reply:
Probably not, but for now I would leave it and for the next version, when also the specs might have some more structure we can improve it based on that.